pgjdbc/src/main/java/org/postgresql/largeobject/LargeObject.java

```diff
@@ -210,12 +216,10 @@ public byte[] read(int len) throws SQLException {
    */
   public int read(byte[] buf, int off, int len) throws SQLException {
     byte[] b = read(len);
```

Comment:
same comment about buf naming, dest is better and makes more sense later in the code.